### PR TITLE
Simplify MANIFEST.in for static files

### PR DESCRIPTION
**Pull request by fao89**
ref: https://www.reddit.com/r/Python/comments/40s8qw/simplify_your_manifestin_commands/

No-Issue

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -48,6 +48,7 @@ ignore = [
     "dev/**",
     "docker/**",
     "openshift/**",
+    "galaxy_ng/app/static/**",
```
Comment (fao89) [inline pyproject.toml:51]: @rochacbruno I mistakenly said you didn't need this, but I was wrong, sorry about that!
checkmanifest compares the package with the VCS, and the gitignore makes them diverge, so indeed you need to ignore it here

**Comment by fao89** [comment]
pulp-operator CI broke because the static files weren't packaged:
https://github.com/pulp/pulp-operator/runs/3234689853?check_suite_focus=true#step:10:3163

File: MANIFEST.in [hunk 2]
```diff
@@ -5,4 +5,4 @@ include unittest_requirements.txt
 include pyproject.toml
 include requirements/requirements.common.txt
 include galaxy_ng/app/webserver_snippets/*
-recursive-include galaxy_ng/app/static/*
+graft galaxy_ng/app/static
```
Comment (rochacbruno) [inline MANIFEST.in:8]: TIL: I was not aware about graft. the `webserver_snippets/*` could be a graft also?

**Review by rochacbruno — APPROVED** [review]

File: MANIFEST.in (hunk 2 above)
Comment (fao89) [reply MANIFEST.in:8]: yes, just replaced it to use graft

**Comment by rochacbruno** [comment]
@fao89 waiting appSre CI to run.

**Comment by himdel** [comment]
:+1: Fixes `worker_1` failing with `ValueError: ("Missing 'Version:' header and/or PKG-INFO file at path: /src/galaxy_ng/galaxy_ng.egg-info/PKG-INFO", galaxy-ng 4.4.0.dev0 (/src/galaxy_ng))` for me :)

**Pull request merged**